addons/web/static/src/js/core/utils.js

```diff
@@ -320,6 +320,8 @@ var utils = {
      * @returns {string}
      */
     human_number: function (number, decimals, minDigits, formatterCallback) {
+        var sign = Math.sign(number);
+        number = Math.abs(number);
```

Comment:
negative numbers of magnitude number >= 21 are no longer correctly formatted with your change. You should do this line 343 (just before the for loop).